src/routes/host/HostList.less

```diff
@@ -21,3 +21,7 @@ a {
     background-color: #27AE5F !important;
```

Comment:
How about making backing image percentage more obvious?
e..g

```
.ant-progress-status-success .ant-progress-success-bg {
    background-color: #1890ff !important;
  }
```

<img width="1496" alt="Screenshot 2024-12-05 at 4 25 25 PM" src="https://github.com/user-attachments/assets/0566df70-4e27-4756-ae1c-b1d6bbb8218e">

Reply:
How about "orange + green"?

Reply:
The example of `orange + green`.
![Screenshot 2024-12-06 at 10 01 00 AM (2)](https://github.com/user-attachments/assets/b2f076c8-2e6f-4152-b3d3-a4d1d9aa1472)

Reply:
I personally prefer `light green + green`. While it may not stand out at first glance, it ensures consistency in the page's overall style, but I don't have a strong opinion on this.
@ChanYiLin Do you have a preference for either option?

Reply:
> light green

I feel it is too light.  How about this one?
#00cc99
![image](https://github.com/user-attachments/assets/d6f5b8f7-526a-46fe-bf80-e27e421fbe44)

Reply:
I prefer not to use "orange + green"
because in the "Dashboard" we already have this kind of Yellow/Orange color for "Reserved"
![Screenshot from 2024-12-06 16-56-44](https://github.com/user-attachments/assets/8ab7ca52-e788-439c-abb4-fcbbcd427b01)
How about a deeper green close to blue, it also fits "Backing Image" concept XD
Just randomly pick one, but maybe can adjust it a little bit.
![Screenshot from 2024-12-06 17-01-03](https://github.com/user-attachments/assets/59a0c689-5614-4b68-9220-73612d5e6600)

Reply:
My bad. I missed the word `not`, lol.
Looks like we agreed on `dark green + green`, I've updated the color in commit https://github.com/longhorn/longhorn-ui/pull/821/commits/b908eb8e2f82c482e965fd461b00143e18b68583. It's 10% darker than the green bar. We can adjust the value if the contrast is not distinct enough.
<img width="1512" alt="Screenshot 2024-12-09 at 9 42 24 AM" src="https://github.com/user-attachments/assets/a159d1c8-b52b-483b-8586-c22a96090102">